default.nix

```diff
@@ -0,0 +1,36 @@
+with import <nixpkgs> { config = {}; overlays = []; };
```

Comment:
This is now even less overridable. I would suggest making it parametrizable like here: https://gitlab.gnome.org/GNOME/buoh/-/blob/f37746cc3161f4b5fd4e9c88b725dfe2977cc400/default.nix#L24-33